base/gpu/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04/release-notes.md

```diff
@@ -1,11 +1,23 @@
 -  **Image Name**: docker pull mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.1-cudnn8-ubuntu20.04 :
 
+:20220714.v1
+------------------- 
+-   **Dependencies** :
+
+-   [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1) 
+-   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5446-1)
+-   [CVE-2022-1304](https://ubuntu.com/security/notices/USN-5464-1) 
+-   [CVE-2022-2068](https://ubuntu.com/security/notices/USN-5488-1) 
+-   [CVE-2022-32208](https://ubuntu.com/security/notices/USN-5495-1)
+
+- released images to mitigate GnuPG, PCRE, dpkg, OpenSSL,curl, libcurl3-nss, e2fsck-static, e2fsck-static, gnupg2, gnupg, libdpkg-perl, gpg, libpcre3, and E2fsprogs Vulnerabilities
+
 :20220708.v1
 ------------------- 
 -   **Dependencies** :
 
 -   [CVE-2019-20838](https://ubuntu.com/security/notices/USN-5425-1) 
--   [CVE-2022-1664](https://ubuntu.com/security/notices/USN-5446CVE-2022-1304-1)
```

Comment:
why its look like this i think ...please check the cve and add like other comments